db/models/applications.py

```diff
@@ -1,5 +1,6 @@
 import random
 import string
+from typing import ParamSpec
```

Comment:
This looks unused to me? Did you run `pre-commit`?

Reply:
It does look unused, not sure how pre-commit did not catch this but have removed now 👍🏾